test/plugin-manager.js

```diff
@@ -204,6 +212,43 @@ describe('Plugin manager', function () {
     manager.initialize();
   });
 
+  before(() => {
```

Comment:
Do we need/want to scope this in its own `describe` block so the `before` and `after` functions don't interact with other tests?